processing_handler.go

```diff
@@ -370,8 +371,9 @@ func handleProcessing(reqID string, rw http.ResponseWriter, r *http.Request) {
 	checkErr(ctx, "timeout", router.CheckTimeout(ctx))
 
 	if originData.Type == po.Format || po.Format == imagetype.Unknown {
-		// Don't process SVG
-		if originData.Type == imagetype.SVG {
+		// Skip processing svg with unknown destination imageType
+		// if it's not forced by AlwaysRasterizeSvg option
+		if originData.Type == imagetype.SVG && !config.AlwaysRasterizeSvg {
```

Comment:
It's better to check `po.SkipProcessingFormats` here instead of duplicating the whole block further

```suggestion
		shouldSkip := slices.Contains(po.SkipProcessingFormats, originData.Type)

		// Skip processing svg with unknown destination imageType
		// if it's not forced by AlwaysRasterizeSvg option
		if originData.Type == imagetype.SVG && (!config.AlwaysRasterizeSvg || shouldSkip) {
```

Reply:
It's not quite right condition and it fails even with old tests, but i try to rewrite it. 
However there are still duplicating with svg case
```golang
if originData.Type == imagetype.SVG && !config.AlwaysRasterizeSvg || shouldSkipProcessing {
			if originData.Type == imagetype.SVG {
```
We have two cases, svg origin data with disabled AlwaysRasterizeSvg and skipProcessing with any origin data. So our condition should work only in both cases
And then we should sanitize svg, so we have to put another check on svg and skip other formats

Reply:
This is a valid condition, you just didn't use it the way I meant. But that's ok